medicinal_plants/lib/src/screens/collection-screen.dart

```diff
@@ -0,0 +1,118 @@
+// ignore_for_file: file_names
+
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class CollectionScreen extends StatelessWidget {
+  final List<Map<String, String>> items = List<Map<String, String>>.generate(
+    20,
```

Comment:
we can move this to data, or much better is

commons/model/plant.dart
commons/data/plant_data.dart